namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-staging/resources/elasticache.tf

```diff
@@ -0,0 +1,28 @@
+module "laa-legal-adviser-api-celery-broker" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=3.1"
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "${var.team_name}"
+  application            = "${var.application}"
+  is-production          = "${var.is-production}"
+  node_type              = "cache.t2.micro"
+  environment-name       = "${var.environment-name}"
+  infrastructure-support = "${var.infrastructure-support}"
+
+  providers = {
+    aws = "aws.london"
+  }
+}
+
+resource "kubernetes_secret" "celery-broker" {
+  metadata {
+    name      = "laa-legal-adviser-api-celery-broker-${var.environment-name}"
+    namespace = "${var.namespace}"
+  }
+
+  data {
+    url                      = "redis://:${module.laa-legal-adviser-api-celery-broker.auth_token}@${module.laa-legal-adviser-api-celery-broker.primary_endpoint_address}:6379"
```

Comment:
I'm curious whether this `url` will work. I tried with our version of Django and Celery and it doesn't support `rediss://` (SSL). Would this work?